org.openandroidinstaller.OpenAndroidInstaller.json

```diff
@@ -0,0 +1,165 @@
+{
+  "app-id": "org.openandroidinstaller.OpenAndroidInstaller",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "22.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "openandroidinstaller",
+  "finish-args": [
+    "--share=ipc",
+    "--share=network",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--device=all"
```

Comment:
Why do you need `--share=network` and `--device=all`?

Reply:
OpenAndroidInstaller crashes without network access https://github.com/openandroidinstaller-dev/openandroidinstaller/pull/144#issuecomment-1527186632

and it also has to access the android devices.

Reply:
Do I understand correctly that at this point Python packages are to be downloaded from https://files.pythonhosted.org/ and it needs the network connection for this?

Reply:
So the connection to the smartphone comes through `--device=all`?

Reply:
Yes it does :)